doc/classes/CodeEdit.xml

```diff
@@ -144,6 +144,12 @@
 				Code regions are delimited using start and end tags (respectively [code]region[/code] and [code]endregion[/code] by default) preceded by one line comment delimiter. (eg. [code]#region[/code] and [code]#endregion[/code])
 			</description>
 		</method>
+		<method name="delete_lines">
+			<return type="void" />
+			<description>
+				Deletes all lines that are selected or have a caret on them.
+			</description>
+		</method>
```

Comment:
I feel like these four methods are common enough to be exposed for CodeEdit, since they're common features in many IDEs. But, I can unexpose them if we want to wait before commiting to them.
Also, it seems that @Paulb23 wants all public methods to be exposed https://github.com/godotengine/godot/pull/86978#issuecomment-1904834139